docs/requirements.md

```diff
@@ -1,26 +1,26 @@
-# Requirements for a technical solution
+# Goals and Requirements for a Technical Solution
 
-## High level requirements: 
+## Goals
 
-To ensure that all users without discrimination will be able to reserve their rights, the solution shall:
+To ensure that all users without discrimination will be able to reserve TDM Rights, the solution shall:
 
-1. Rely on open standards and be interoperable.
-1. Rely on tools and formats widely supported by web crawlers.
-1. Be easy to implement, with no economic or technical barriers.
+1. Rely on open standards  - especially Web standards - and be interoperable.
+1. Rely on tools and formats widely supported by publishers and TDM Actors.
```

Comment:
We don't have a definition for _publisher_ - so either we need to define it *or* not use it.

Reply:
It's difficult to replace it by rightsholder, as rightsholders often rely on other parties to publish their content on the web.
So ok let's define it (see update of the Vocabulary).